EIPS/eip-3777.md

```diff
@@ -0,0 +1,312 @@
+---
+eip: 3777
+title: Time proof non-fungible Token
+author: Cyberforker(cyberforker@outlook.com)
+discussions-to: https://ethereum-magicians.org/t/a-time-proof-non-fungible-token/
+status: draft
+type: Standards Track
+category: ERC
+created: 2021-09-01
+requires: 165, 721
+---
+## Simple Summary

```

Comment:
```suggestion
## Simple Summary
```
This section is no longer valid, please use the `description` preamble field. (see the `eip-template.md` for more info)